src/domain/customers/tableTools/customerMessageForm.module.scss

```diff
@@ -0,0 +1,33 @@
+@import 'spacings';
+
+.form {
+  > * {
+    margin: units(1) 0;
+  }
```

Comment:
@ahie defined a mixin that might come in handy here: 
https://github.com/City-of-Helsinki/berth-reservations-admin/blob/d73fc3ac6f15b65080f1e3eb6ec8f9f7d59faa7e/src/assets/styles/_spacings.scss#L44

Reply:
Unfortunately I don't have a common classname for the child elements in this case, so I cannot use that mixin.